pom.xml

```diff
@@ -94,7 +94,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/x
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
     <!-- maven plugin versions -->
-    <maven-deploy-plugin.version>2.8.1</maven-deploy-plugin.version>
+    <maven-deploy-plugin.version>3.1.1</maven-deploy-plugin.version>
```

Comment:
Rather than keep a diff in the pom, you can just pass in `-Dmaven-deploy-plugin.version=3.1.1`. This way we still have a chance of encapsulating all of these build-related modifications in a buildpack outside the repo

Reply:
Good call, I can adjust that.